gallery/gallery/pubspec.yaml

```diff
@@ -35,6 +36,7 @@ dev_dependencies:
 
 flutter:
   assets:
+    - fonts/google_fonts/
```

Comment:
Why is this needed?

Reply:
The google_fonts package will first check assets for font files before fetching them and caching them. See https://github.com/material-foundation/google-fonts-flutter/blob/1255da8321fb84be2ffe56fcc5f792eb1dadf83d/README.md#bundling-font-files-in-your-applications-assets for more details

Reply:
We could chose not to bundle them, but then you'd get a flicker as fonts load the first time, and the fonts wouldn't load if you opened the app offline for the first time.

Reply:
Gotcha, the readme mentions that the directory should be `- google_fonts/`, but I guess ours works with the extra level in there? Or should we modify that too

Reply:
It can use any directory. I just figured it made more sense to put them under fonts/